### PR TITLE
adding carousel item for forecast dashboard

### PR DESCRIPTION
**Pull request by kateharwood**

### CODE REVIEW

**Comment by Letsgohyde** [comment]
hero image 1x
image

Hero image 2x
image

Link to figma mock: https://www.figma.com/file/fDDK26m1S1rqhxFtk9edr5/v2_DELPHI_VisD-UI-Kit?node-id=402%3A2646

@kateharwood here you go :)

**Comment by Letsgohyde** [comment]
I focused the visible area on the chart and key below it. Since the left panel was mostly covered by the overlay I thought this might focus the user on the cool line chart comparisons that are possible. If that works for you?

File: content/_index.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ carousel:
     ref: covidcast
     alt: Explore COVIDcast
     image: covidcast_withfill.jpg
+  - pre: Our Tools
+    title: Forecast Evaluation Dashboard
+    ref: forecast-eval
```
Comment (kateharwood) [inline content/_index.md:19]: @sgratzl is there somewhere I need to define "forecast-eval" ref so that it points to {currentUrl}/forecast-eval?

Comment (sgratzl) [reply content/_index.md:14]: please use the link version instead, e.g., `link: /forecast-eval` refs are internal hugo references.

Comment (kateharwood) [reply content/_index.md]: Thanks! Ready for review now

**Comment by sgratzl** [comment]
please also add a `.webp` version of the image

**Comment by sgratzl** [comment]
and a resolution of 1440x500 is enough

**Review by tildechris — APPROVED** [review]
LGTM.

**Review by sgratzl — CHANGES_REQUESTED** [review]
see comments before regarding image types and sizes

**Comment by kateharwood** [comment]
@sgratzl done

**Review by sgratzl — APPROVED** [review]
👍

**Pull request merged**